cmds/core/ip/link_linux.go

```diff
@@ -23,7 +23,7 @@ const linkHelp = `Usage: ip link add  [ name ] NAME
 
 	ip link delete { DEVICE | dev DEVICE  } 
 
-	ip link set { DEVICE | dev DEVICE | group DEVGROUP }
```

Comment:
I see that we have `setLinkGroup` function but is not used by anyone? Do you plan to remove it or add additional functionality latter?

Reply:
Yes, you are right. I pushed a commit, which correctly adds the functionality.